_pages/algorithm-development/your-first-algo.md

```diff
@@ -13,216 +13,109 @@ redirect_from:
+To access Algorithmia's built-in Web IDE, click the "Web IDE" button in the modal, or click the **Source Code** tab from your newly created algorithm's homepage if the modal is no longer shown.
 
 <img src="{{site.cdnurl}}{{site.baseurl}}/images/post_images/algo_dev_lang/generic_algorithm_description.png" alt="Algorithm descrption page" class="screenshot img-md">
 
-Notice the tabs: **Overview**, **Docs**, **Discussion**, **Builds**, **Versions**, **Settings**, and **Source Code**.
-
-The **Overview** tab displays your algorithm's basic information, statistics, and settings. You can add a short summary stating what your algorithm is and why people might be interested in it (for example how it solves a particular problem for a specific use case). On this tab, users can run your algorithm with the default input that you provide during the algorithm-publishing step (see below in this guide), or test out your algorithm with their own input. Below this on the page is information on how to call your algorithm in multiple programming languages from outside of the Algorithmia platform.
-
-In the **Docs** tab you will want to add information to explain to users how to use your algorithm, including complete information about the input type(s) allowed and what the expected output(s) will be. The embedded README file supports Markdown syntax and should be edited and updated if the usage of the algorithm changes.
-
-The **Discussion** tab displays any comments and questions that users have posted about your algorithm, so that you can keep up to date with user feedback and provide responses as necessary.
+In the Web IDE, you'll see that your algorithm comes with some template code: a function called `apply()` that takes a string as input and returns "Hello" concatenated with that string.
 
-Under the **Builds** tab you can view release notes and permissions associated with each published version of the algorithm.
+On Algorithmia, every algorithm is required to have an `apply()` function—it defines the algorithm's entry point and serves to standardize algorithms across the platform regardless of programming language. Authors are encouraged to leverage this standardization and to design their algorithms in a way that makes them predictable for end users and that supports algorithm pipelining (chaining multiple algorithms together).
 
-Under the **Versions** tab you can view information about all of the discreet algorithm builds that have occurred, including specific Git hashes and the version number associated with each.
+In order to run the algorithm, you'll first need to compile (build) it by clicking the "Build" button on the top-right corner of the source code editor. You'll need to complete this step even for an interpreted language such as Python, because we still need to package up all the source code into a callable algorithm instance on the back end.
```

Comment:
+1 for describing why this needs to happen 👏

Reply:
👍🏼